SpatialGDK/Documentation/content/glossary.md

```diff
@@ -1,33 +1,397 @@
+
+A SpatialOS entity can have as many components as you like, but it must have at least [`Position` (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#position) and [`EntityAcl`](#access-control-list-acl). Most entities will have the [`Metadata` (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#metadata) component.
+SpatialOS components are defined as files in your [schema](#schema).
+[Entity access control lists](#access-control-list-acl) govern which workers can [read from](#read-access) or [write to](#write-access) each component on an entity.
+
+> Related:
+>
+> * [Designing components (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/design/design-components)
+> * [Component best practices (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/design/component-best-practices)
+> * [Introduction to schema (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/schema/introduction)
+
+### SpatialOS entity
+All of the objects inside a [SpatialOS world](#spatialos-world) are SpatialOS entities: they’re the basic building blocks of the world. Examples include players, NPCs, and objects in the world like trees. A SpatialOS entity approximates to an Unreal Actor.
+SpatialOS entities are made up of [SpatialOS components](#spatialos-component), which store data associated with that entity.
+[Workers](#workers) can only see the entities they're [interested in](#interest).
+
+> Related:
+>
+> * [SpatialOS Concepts:Entities (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/concepts/world-entities-components#entities)
+> * [Designing SpatialOS entities (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/design/design-entities)
+
+### SpatialOS Runtime
+
+>Not to be confused with the [SpatialOS world](#spatialos-world).
+
+Also sometimes just called “SpatialOS”. 
+A SpatialOS Runtime instance manages the [SpatialOS world](#spatialos-world) of each [deployment](#deployment) by storing all [SpatialOS entities](#spatialos-entity) and the current state of their [SpatialOS components](#spatialos-components). [Workers](#workers) interact with the SpatialOS Runtime to read and modify the components of an entity as well as send messages between workers.
+
+### SpatialOS SDK
+This is a set of low-level tools in several programming languages which you can use to integrate your game project with SpatialOS. It consists of the Worker SDK (or “Worker module”) and Platform SDK (or “Platform module”). 
+If you are using Unreal, you do not need to use the Worker SDK or Platform SDK, however, you can use the Worker SDK to extend or compliment the functionality of the GDK for Unreal. 
```

Comment:
Add line break above.

Also:

- Unreal -> the GDK for Unreal?
- compliment -> complement